pkg/pool-weighted/contracts/smart/InvestmentPool.sol

```diff
@@ -159,7 +199,9 @@ contract InvestmentPool is BaseWeightedPool, ReentrancyGuard {
     }
 
     function _getMaxTokens() internal pure virtual override returns (uint256) {
-        return _MAX_WEIGHTED_TOKENS;
+        // The upper bound is WeightedMath.MAX_WEIGHTED_TOKENS, but this is constrained by other factors, such as Pool
+        // creation gas consumption (which is linear).
+        return 50;
```

Comment:
```suggestion
        return _INVESTMENT_MAX_TOKENS;
```

Can we pull this out to a constant at the top of the contract? It'll make it easier to tell the maximum number of tokens at a glance.